doc/develop/test/ztest_deprecated.rst

```diff
@@ -1,266 +0,0 @@
-.. _test-framework-deprecated:
-
-ZTest Deprecated APIs
-#####################
-
-Ztest is currently being migrated to a new API, this documentation provides information about the
-deprecated APIs which will eventually be removed. See :ref:`Test Framework <test-framework>` for the new API.
-Similarly, ZTest's mocking framework is also deprecated (see :ref:`Mocking via FFF <mocking-fff>`).
```

Comment:
`ZTest's mocking framework is also deprecated`
Is it still so ? and if it is, then shouldn't this note be somewhere else ?

Reply:
obviously it is not deprecated or was not deprecated correctly, we do not need to add this anywhere else, this document was a left-over and should be remove regardless. Deprecating legacy ztest mocking will need to be done individually.